### DISCUSSION TITLE
Panels don't auto-refresh with changes with OpenLens?

### DISCUSSION THREAD

**Discussion by toast-gear** [MuhammedKalkan/OpenLens#71] [Q&A]
Just wanted to ask this here in case it's just me, I understand OpenLens is just Lens stripped back to the opens source bits only and not something custom so I'm not expecting a fix or anything really.

I'm finding that panels don't auto-refresh meaning I need to change panel and come back or reload the application `cmd +r`? For example if I delete a pod the pod will visually stay as terminating until I force a refresh of the panel by switching panels and returning or hitting `cmd + r`. Is this something everyone is experiencing? Just a bit surprised I haven't seen anyone mention it as it does make using OpenLens cumbersome.

**Environment**
OpenLens Version v6.2.5
ARM M2 Mac

**Comment by ebbypeter** [comment]
I believe it is the same case with the Lens IDE version as well. Or at least it was until I moved to Open Lens.
You will need to refresh or move from the panel to another and come back to see the changes.

**Reply by toast-gear** [reply]
That's not the case for me at least with regular Lens and I'm pretty sure it never has been. If I delete a pod for instance, once it's terminated it disappears without me having to change panel or anything. If it's backed by a deployment or something a new one will be provisioned which also appears without having to change view or anything.
